src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container *ngIf="gatewayGroup$ | async as gateways">
+  <cd-table
+    #table
+    [data]="gateways"
+    [columns]="columns"
+    columnMode="flex"
+    selectionType="single"
+    identifier="name"
+    (updateSelection)="updateSelection($event)"
+    (fetchData)="fetchData()">
+  </cd-table>
+</ng-container>
+
+<ng-template #dateTpl
+             let-events="data.value">
+  <span *ngIf="events?.length">{{ events[0].created | date:'EEE d MMM, yyyy' }}</span>
+</ng-template>
+<ng-template #gatewayTpl
+             let-gateway="data.value">
+  <div class="gateway-cell">
+
+    <span class="status-block">
```

Comment:
rather than defining our own custom css which is going against carbon or not fully utilizing carbon as it is, we need to use the carbon components. For instance I saw there is this [stack](https://angular.carbondesignsystem.com/?path=/docs/layout-stack--docs), in carbon that does the same thing.